### PR TITLE
added user authentication feature with login and logout button and re…

### PR DESCRIPTION
**Pull request by javidraeisy**
…act router

### CODE REVIEW

File: wild-swimming-app/src/App.tsx [hunk 1]
```diff
@@ -1,12 +1,25 @@
+// import React from "react";
 import "./App.css";
-import Header from "./pages/home/Homepage";
+// import { Link } from "react-router-dom";
```
Comment (amnioc) [inline wild-swimming-app/src/App.tsx:3]: commented out code to me removed if not in use :)

File: wild-swimming-app/src/App.tsx [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+// import Header from "./components/Header/Header";
+import RootLayout from "./components/RootLayout/RootLayout";
```
Comment (amnioc) [inline wild-swimming-app/src/App.tsx:5]: is this user authentication? could something be added to help somoene know what the link is when they pick up the code?

File: wild-swimming-app/src/components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+// import React from "react";
+// import { Link } from "react-router-dom";
+import "./styles.css";
+import { useAuth0 } from "@auth0/auth0-react";
+import LoginButton from "../loginbutton/loginbutton";
+import LogoutButton from "../logoutbutton/logoutbutton";
```
Comment (amnioc) [inline wild-swimming-app/src/components/Header/Header.tsx:6]: excited to see this live!

File: wild-swimming-app/src/App.tsx (hunk 2 above)
Comment (javidraeisy) [reply wild-swimming-app/src/App.tsx:5]: The RootLayout is part of auth0Provider and it is included in the app.tsx in the RouterProvider here I used reactRouter to help making different web pages and the RootLayout is just required settings for using auth0

**Pull request closed without merging**